ui/app/AppLayouts/Profile/popups/TokenListPopup.qml

```diff
@@ -56,7 +57,18 @@ StatusDialog {
 
             CustomHeaderDelegate {}
         }
-        delegate: CustomDelegate {}
+        delegate: CustomDelegate {
+            width: contentItem.width
+            height: 64
+
+            name: model.name
+            image: model.image
+            chainName: model.chainName
+            symbol: model.symbol
+            address: model.address
+            explorerUrl: model.explorerUrl
+            isTest: model.isTest
```

Comment:
I wonder why you need suddenly all those properties spelled out; doesn't the builtin `model.foo` no longer work here?

Reply:
Yeah I see... it's a good practice to set it explicitly